include/DREAM/Equations/Scalar/ConstantSPIVelocityPositionTerm.hpp

```diff
@@ -46,12 +50,29 @@ namespace DREAM {
                 this->initial_data=xp_init;
             }
         }
+        void SetTimeDelay(const real_t *t_delay, bool copy=true){
```

Comment:
This file seems to be containing more method implementations than other declarations. It would be better to separate it into a ``.hpp`` containing declarations and a ``.cpp`` containing the method implementations (constructor, destructor, ``GetData()``, ``SetTimeDelay()`` and ``Rebuild()``).